shared-bindings/synthio/LFO.c

```diff
@@ -74,10 +74,10 @@ static const uint16_t triangle[] = {0, 32767, 0, -32767};
 //|         scale: BlockInput = 1.0,
 //|         offset: BlockInput = 0.0,
 //|         phase_offset: BlockInput = 0.0,
-//|         once=False,
-//|         interpolate=True
-//|     ):
-//|         pass
+//|         once: bool = False,
+//|         interpolate: bool = True,
+//|     ) -> None:
+//|         """Initialize an instance of the class."""
```

Comment:
Is either that or adding `pass`/`...`, otherwise we get invalid syntax due to function having no body. Thought a generic comment was slightly better 😅 
```py
def foo(
    bar: baz,
) -> None:

def next_docstring_line():
```

Reply:
I think `pass` is better for `__init__` because the class level comment is used for it too.